incubator/hnc/plugin/hnc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
```

Comment:
Do you happen to know where other projects put their krew YAMLs? I was wondering if it should go in `cmd/kubectl` or maybe `config/kubectl` instead?

Reply:
I went through first 20 repos on [krew-index](https://github.com/kubernetes-sigs/krew-index) list and found only 3 put krew.yml into their repos (and there is no exact path). AFAIK, once we have releases, we dont need krew yml in the repo instead put it into [krew-index](https://github.com/kubernetes-sigs/krew-index) submitted as a PR. Any changes made e.g version release will revised on other PR.

Reply:
Ok in that case (if this typically isn't included in the repo), let's put it in `hack/` and add a comment saying that it will eventually move to `krew-index` instead - sgty?

Reply:
sgtm. Updated